### PR TITLE
Update CHANGELOG.md for 10.2.0.1

### PR DESCRIPTION
**Pull request by folmos-at-orange**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,20 @@
   - Example: 10.2.1.4 is the 5th version that supports khiops 10.2.1.
 - Internals: Changes in *Internals* sections are unlikely to be of interest for data scientists.
 
-## 10.2.0.1 - 2023-02-15
+## 10.2.0.1 - 2024-03-21
+
+### Added
+- (`sklearn`) 1:1 relations to multi-table datasets.
+- (`sklearn`) Estimators' `fit` method now accept single-column pandas dataframes as `y` target.
```
Comment (popescu-v) [inline CHANGELOG.md:13]: s/method/methods/ (because there are several estimators)

**Review by popescu-v — APPROVED** [review]
LGTM, except for a typo.

File: CHANGELOG.md (hunk 1 above)
Comment (folmos-at-orange) [reply CHANGELOG.md:13]: Ok

**Review by popescu-v — APPROVED** [review]
LGTM

**Pull request merged**